QueryGenie/AttributeProtocol.swift

```diff
@@ -535,7 +535,7 @@ public func << <A: AttributeProtocol, V>(left: A, right: [V]) -> NSComparisonPre
     return left.isIn(right)
 }
 
-public func << <A: AttributeProtocol, V: Comparable>(left: A, right: Range<V>) -> NSComparisonPredicate where A.ValueType == V {
+public func << <A: AttributeProtocol, V>(left: A, right: Range<V>) -> NSComparisonPredicate where A.ValueType == V {
```

Comment:
Why does `V` not need to conform to `Comparable` anymore? Though we are converting to `NSObject` in the `isInBetween` function, if they objects don't conform to `Comparable` the behavior for their comparison will be undefined, won't it?

Reply:
I think you're right. This was changed in commit dfed2ae. Am I wrong in thinking that when you made the xcode-9 branch you just spiked it? I, mistakenly, assumed that those changes were good to go. I will revisit those assumptions.